app/styles/components/_footer.scss

```diff
@@ -7,6 +7,8 @@ $svg-height-mobile: 16px;
 
 $icon-size: 25px;
 
+$footer-break-navlinks: 940px;
```

Comment:
I'd rather keep it as is or changing it to just 'footer_break', it is a specific case for the current footer width. Renaming it 'mobile' or 'tablet' could be misleading as it has nothing to do with being on a different device, it's screen-size related;